lib/stream.js

```diff
@@ -21,10 +21,17 @@
 
 'use strict';
 
+const {
+  ObjectDefineProperty,
+} = primordials;
+
 const pipeline = require('internal/streams/pipeline');
```

Comment:
In here, the `pipeline` and `finished` functions should expose custom promisify symbols that return the same version of the function from `streams/promises` such that...

```js
const { pipeline, finished } = require('streams/promises');
const streams = require('streams');
const { promisify } = require('promisify');

assert.strictEqual(pipeline, promisify(streams.pipeline));
assert.strictEqual(finished, promisify(streams.finished));
```

Reply:
fixed